lib/paths-cache.js

```diff
@@ -104,7 +104,7 @@ export default class PathsCache extends EventEmitter {
     if (this.config.excludeVcsIgnoredPaths) {
       this._repositories.forEach((repository) => {
         if (ignored) {
-          return
+          return true
```

Comment:
why is this  changed?

Reply:
Because logically, from the function name `_isPathIgnored`, it returns whether "the path is to be ignored"

But it returns nothing when `ignored` variable is true, 
Doesn't that mean it's false, or "the path is NOT ignored" ?

Sorry if I misunderstand the function

Reply:
the `return` returns from the `forEach` anonymous function not `_isPathIgnored`.

Reply:
Oh okay, sorry
This is my first PR, can you help/tell me how to cancel that commit?

Reply:
run `git reset --soft HEAD~1 && git push -f`

Reply:
Ok done, thank you